docs/index.md

```diff
@@ -54,28 +54,36 @@ that include, but are not limited to, the following:
-   binary must be built as a separate project and used by validators. This is how the Polkadot relay chain is built and
-   should suffice for almost all needs in the near future. For a tutorial on this, see
-   [creating your first Substrate chain](tutorials/create-your-first-substrate-chain/index.md).
-
-3. **With the Substrate Core**: The entire [FRAME](knowledgebase/runtime/frame.md) can be ignored, and the entire
-   runtime can be designed and implemented from scratch. This could be done in _any language_ that can target
-   WebAssembly. If the runtime can be made to be compatible with the abstract block authoring logic of the Substrate
-   node, then you can simply construct a new genesis block from your Wasm blob and launch your chain with the existing
-   Rust-based Substrate client. If not, then you will need to alter the client's block authoring logic, and potentially
-   even alter the header and block serialization formats. In terms of development effort, this is by far the most
-   difficult way to use Substrate, but also gives you the most freedom to innovate.
+1. **With the Substrate Node**: You can run the pre-designed
+   [Substrate Node](https://github.com/paritytech/substrate/tree/master/bin/node) and
+   [configure](https://github.com/paritytech/substrate/blob/master/bin/node/cli/src/chain_spec.rs) it with a genesis
+   block that includes the default node
+   [runtime](https://github.com/paritytech/substrate/blob/master/bin/node/runtime/src/lib.rs). In this case, you just
+   need to configure a JSON file and launch your own blockchain. This affords you the least amount of customization,
+   only allowing you to change the genesis parameters of the included runtime modules such as: Balances, Staking, and
+   Sudo. For a tutorial on doing this, see
+   [Start a Private Network with Substrate](tutorials/start-a-private-network/index.md).
+
+2. **With Substrate FRAME**: You can easily create your own custom runtime using [FRAME](knowledgebase/runtime/frame.md)
+   (Framework for Runtime Aggregation of Modularized Entities), which is the method used by the Substrate Node. This
+   affords you a very large amount of freedom over your blockchain's logic, and allows you to configure data types,
+   select from a library of modules (called "pallets"), and add your own custom pallets. Much can be changed without
+   touching the block-authoring logic since it is directed through runtime logic. If this is sufficient, then the
+   existing Substrate binary can be used for block authoring and syncing; the
+   [Substrate Developer Hub Node Template](https://github.com/substrate-developer-hub/substrate-node-template) is a
+   helpful starting point for projects with such needs. If the block authoring logic needs to be modified, then a new
+   block-authoring binary must be built as a separate project and used by validators. This is how the
```

Comment:
This bit about block authoring seems too in-the-weeds to me. Basically this is a list of three ways to get the runtime you want. Dipping into block authoring is kind of an orthogonal thing. And even if you do want to keep it it isn't correct that the authoring would happen in a separate binary.

Reply:
Oh I see. You mean the binary with modified authoring logic would be separate from the node template? That's true.